src/Components/GetArticleComments.jsx

```diff
@@ -0,0 +1,43 @@
+import { useParams } from "react-router-dom";
+import { useState,useEffect } from "react";
+import { getSingleArticleComments } from "../utility/api";
+const GetArticleComments = () => {
+const [comments, setComments] = useState(null)
+const [isLoading, setIsLoading] = useState(true)
+const { article_id } = useParams()
+useEffect(() => {
+    if (article_id) {
+        getSingleArticleComments(article_id).then((data) => {
+        setComments(data)
+        setIsLoading(false)
+      }).catch((err) => {
+        console.error(err)
+        setIsLoading(false)
+      });
+    }
+  }, [article_id]);
```

Comment:
Because you're getting this from params, this will never change and doesn't need to be here. You need the array but not necessarily any elements